pkg/server/admin_test.go

```diff
@@ -346,6 +346,46 @@ func TestAdminAPIDatabaseSQLInjection(t *testing.T) {
 	}
 }
 
+func TestAdminAPINonTableStats(t *testing.T) {
+	defer leaktest.AfterTest(t)()
+	testCluster := serverutils.StartTestCluster(t, 3, base.TestClusterArgs{})
+	defer testCluster.Stopper().Stop(context.Background())
+	s := testCluster.Server(0)
+
+	// skipping TableStatsResponse.Stats comparison, since it includes data which aren't consistent (time, bytes)
```

Comment:
Nit: phrase as a sentence (this is a convention we have)
```suggestion
	// Skip TableStatsResponse.Stats comparison, since it includes data which aren't consistent (time, bytes).
```

Reply:
oops, I didn't read convention before pushing PR. Read and fixed :)